### PR TITLE
Openstack exceptions

### PR DESCRIPTION
**Pull request by Danmcaires**

### CODE REVIEW

File: src/api_request.py [hunk 1]
```diff
@@ -71,13 +72,22 @@ def get_path_completion(self):
 
 
     def get_API_response(self):
-        path = self.get_path()
+        # path = self.get_path()
+        path = "self.openstack_sdk.KEYSTONE.services.show('cinder')"
```
Comment (ldeataid) [inline src/api_request.py:76]: I believe this was not intended to be part of the commit, right?

Comment (Danmcaires) [reply src/api_request.py]: Yes, fixed!!

**Comment by ldeataid** [comment]
LGTM

**Review by ldeataid — APPROVED** [review]
LGTM

**Pull request merged**